osmaxx-py/osmaxx/contrib/auth/migrations/0002_add_default_usergroup_osmaxx.py

```diff
@@ -8,7 +8,13 @@
 def update_site_forward(apps, schema_editor):
     """Add group osmaxx."""
     Group = apps.get_model("auth", "Group")
-    Group.objects.create(name=settings.OSMAXX_FRONTEND_USER_GROUP)
+    Permission = apps.get_model("auth", "Permission")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    ExtractionOrder = apps.get_model("excerptexport", "ExtractionOrder")
+    group = Group.objects.create(name=settings.OSMAXX_FRONTEND_USER_GROUP)
+    content_type = ContentType.objects.get_for_model(ExtractionOrder)
+    permission, created = Permission.objects.get_or_create(codename='add_extractionorder', content_type=content_type)
+    group.permissions.add(permission)
```

Comment:
Would be an argument for it, but I think this is just fine: backwards migrations would be much harder when having separate migrations.